feeds/tip/cloud_discovery/files/usr/bin/cloud_discovery

```diff
@@ -78,6 +83,19 @@ function gateway_load() {
 	return readjsonfile('/etc/ucentral/gateway.json');
 }
 
+function discovery_state_write() {
+	let t = localtime();
+	let updated = {
```

Comment:
please just use time()/epoc here. it will make it easier for consumers to just have epoch rather than having to convert/parse a string